osbs/utils.py

```diff
@@ -702,6 +702,19 @@ def should_retry_cb(ex):
     return retry
 
 
+def retry_on_not_found(func):
+    @wraps(func)
+    def retry(*args, **kwargs):
+        # Only retry when OsbsResponseException was raised due to not found
+        def should_retry_cb(ex):
+            return ex.status_code == http_client.NOT_FOUND
```

Comment:
We don't have coverage here according to coveralls, but we do on the equivalent part of `retry_on_conflict`.

Reply:
I will try to add it , I think I have idea

Reply:
idea failed